### ISSUE TITLE
[BUG]: buffering wheel doesn't go away.

### ISSUE THREAD

**Issue by Stryker1869** [yuliskov/SmartTube#3958]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

23.89

### Device Type

Smart TV/Box

### Affected Android

Android TV 11

### Steps to reproduce the bug

1. Switch profiles
2. Watch a video 
3. Switch profiles 

### Actual behavior

Once you switch profiles and watch videos the buffering wheel stays on while videos play. Have to close the app to fix it.

**Comment by Chealer** [comment]
Does this persist with the latest stable version? If so, please clarify how to perform step #1.

**Comment by Stryker1869** [comment]
> Does this persist with the latest stable version? If so, please clarify how to perform step [#1](https://github.com/yuliskov/SmartTube/issues/1).

Yes.  I have 2 different YouTube profiles I switch between. Most of the he time time after I've switched it leaves the buffering wheel indefinitely when you're on the subscriptions.  I hope that clears it up a bit. It's hard to explain.

**Comment by Chealer** [comment]
Thank you @Stryker1869 
When you refer to "YouTube profiles", do you mean Google accounts, or what?

**Comment by Stryker1869** [comment]
> Thank you [@Stryker1869](https://github.com/Stryker1869) When you refer to "YouTube profiles", do you mean Google accounts, or what?

Yes. Different Google accounts

**Comment by Chealer** [comment]
Thanks again. Please indicate which version you use now (if easy) and retitle this more specifically. For example:

> [BUG]: buffering wheel sticks after switching Google account/profile
